### DISCUSSION TITLE
官网居然需要翻墙，我很诧异

### DISCUSSION THREAD

**Discussion by weimingjue** [Tencent/libpag#2023] [General] [answered]
打开官网居然出现了久违的长时间Loading：https://pag.art
准备预览一个试试，结果js直接就下载失败：https://pag.art/player.html
这还是国产的吗？

**Comment by kevingpqi123** [comment, marked as answer]
反馈问题请不要上纲上线，我这边通过手机 5G 打开官网速度很快，是没有问题的

**Reply by weimingjue** [reply]
![screenshot-20240103-095517](https://github.com/Tencent/libpag/assets/25866952/3f15925c-1caa-4238-b7e8-0bf884716128)

**Comment by weimingjue** [comment]
![screenshot-20240103-092921](https://github.com/Tencent/libpag/assets/25866952/91da4aba-b882-438e-9282-cf185eccaa75)
电信宽带
